wiki/Lua-Hyperspace-Module.md

```diff
@@ -36,6 +36,8 @@ All calls are under `Hyperspace`
    - Returns the main instance of [`MouseControl`](#MouseControl). Always use this to access any members and methods belonging to the [`MouseControl`](#MouseControl) class, or the shortcut `Hyperspace.Mouse`.
 - `TextLibrary :GetTextLibrary()`
    - Returns the main instance of [`TextLibrary`](#TextLibrary). Always use this to access any members and methods belonging to the [`TextLibrary`](#TextLibrary) class, or the shortcut `Hyperspace.Text`.
+- `EventGenerator GetEventGenerator()`
```

Comment:
Missing `.` before function name

Reply:
Actually it was a `:` that was missing.

Reply:
Oh whoops I thought it was static, my bad